src/components/Login.tsx

```diff
@@ -16,24 +17,48 @@ const Login = ({ loginWalletOptions }: LoginProps): JSX.Element => {
   const [loading, startLoading] = React.useState<boolean>(false);
   const [alertError, setAlertError] = React.useState<string>("");
   const [popoverVisible, setPopoverVisible] = React.useState<boolean>(false);
+  const [registrationVisible, setRegistrationVisible] = React.useState<boolean>(
+    false
+  );
+
+  const [walletAddress, setWalletAddress] = React.useState<string>("");
+  const [walletType, setWalletType] = React.useState<wallet.WalletType>(
+    loginWalletOptions || wallet.WalletType.NONE
+  );
 
   const dispatch = useAppDispatch();
   const userId = useAppSelector((state) => state.user.id);
-  const walletType = useAppSelector((state) => state.user.walletType);
+
+  const setLoginAndSession = (fromId: string) => {
+    setRegistrationVisible(false);
+    dispatch(userLogin({ id: fromId, walletType }));
```

Comment:
Im curious to learn why we don't also store the walletAddress in Redux?

Reply:
Not sure, perhaps we don't need it once we login and maybe it's a security issue.

Reply:
Mainly because we don't need it. It gets stored in the `signer` in the sdk config so that's how it's used in signing. Everything else is keyed off DSNP id.